Raven.Studio.Html5/App/commands/database/documents/getCollectionLabelCommand.ts

```diff
@@ -0,0 +1,35 @@
+﻿import commandBase = require("commands/commandBase");
+import database = require("models/resources/database");
+
+class getCollectionLabelCommand extends commandBase {
```

Comment:
I think we don't need this command, as we fetch StudioConfig  document in shell. (fetchStudioConfig method). As result we always have globally accessible information about collections name mappings. 

Just modify shell::fetchStudioConfig, parse Raven/StudioConfig document, and save per collection names overrides.